### PR TITLE
LP-2479 Handle ADG admins and send acceptance, waitlist email

### PR DESCRIPTION
**Pull request by ahmed-faraz46**
[LP-2479](https://philanthropyu.atlassian.net/browse/LP-2479) master PR
- Override get_queryset to filter relevant applications
- Send Acceptance, wait list email
- Add one on one relationship in Group and BusinessLine
- Move send_mandrill_email helper in MandrillClient
- Add generic helper get_user_first_name
- Add unit test for acceptance, waitlist email and ADG admins application filter

### CODE REVIEW

File: openedx/adg/lms/helpers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+ADG LMS Helpers
+"""
+
+
```
Comment (asadali145) [inline openedx/adg/lms/helpers.py:5]: I think you have missed to remove these empty lines.

**Review by asadali145 — DISMISSED** [review]
LGTM. Left a comment have a look.

File: openedx/adg/lms/helpers.py (hunk 1 above)
Comment (ahmed-faraz46) [reply openedx/adg/lms/helpers.py:5]: Removed

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1146/workflows/workflows_id/jobs/1146/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (71.4%): Missing lines 67-68
openedx/adg/common/lib/mandrill_client/email_data.py (100%)
openedx/adg/lms/applications/admin.py (26.7%): Missing lines 247-248,250-251,253,304,307,342,346-347,352
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/handlers.py (0.0%): Missing lines 19,23-24
openedx/adg/lms/applications/helpers.py (0.0%): Missing lines 113
openedx/adg/lms/applications/migrations/0006_businessline_group.py (0.0%): Missing lines 3-4,7,9,14
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/rules.py (100%)
openedx/adg/lms/applications/tests/conftest.py (45.5%): Missing lines 59-61,74-75,77
openedx/adg/lms/applications/tests/constants.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (48.3%): Missing lines 81-82,84,104-105,107-108,207,233,241,278-279,281,286-287
openedx/adg/lms/applications/tests/test_helpers.py (50.0%): Missing lines 74
openedx/adg/lms/helpers.py (50.0%): Missing lines 16
openedx/adg/lms/student/helpers.py (14.3%): Missing lines 39,67,92,106,131,155
openedx/adg/lms/student/tests/test_helpers.py (50.0%): Missing lines 55,88,113,130,172,191,209
openedx/adg/lms/tests/constants.py (0.0%): Missing lines 2-3
-------------
Total:   116 lines
Missing: 60 lines
Coverage: 48%
-------------
```

**Review by asadali145 — APPROVED** [review]

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1150/workflows/workflows_id/jobs/1150/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (71.4%): Missing lines 67-68
openedx/adg/common/lib/mandrill_client/email_data.py (100%)
openedx/adg/lms/applications/admin.py (26.7%): Missing lines 247-248,250-251,253,304,307,342,346-347,352
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/handlers.py (0.0%): Missing lines 19,23-24
openedx/adg/lms/applications/helpers.py (0.0%): Missing lines 113
openedx/adg/lms/applications/migrations/0006_businessline_group.py (0.0%): Missing lines 3-4,7,9,14
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/rules.py (100%)
openedx/adg/lms/applications/tests/conftest.py (45.5%): Missing lines 59-61,74-75,77
openedx/adg/lms/applications/tests/constants.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (48.3%): Missing lines 81-82,84,104-105,107-108,207,233,241,278-279,281,286-287
openedx/adg/lms/applications/tests/test_helpers.py (50.0%): Missing lines 74
openedx/adg/lms/helpers.py (50.0%): Missing lines 16
openedx/adg/lms/student/helpers.py (14.3%): Missing lines 39,67,92,106,131,155
openedx/adg/lms/student/tests/test_helpers.py (50.0%): Missing lines 55,88,113,130,172,191,209
openedx/adg/lms/tests/constants.py (0.0%): Missing lines 2-3
-------------
Total:   116 lines
Missing: 60 lines
Coverage: 48%
-------------
```

**Review by emraan004 — APPROVED** [review]
👍

**Pull request merged**